### ISSUE TITLE
Release Issue - Place holder

### ISSUE THREAD

**Issue by hazendaz** [mybatis/spring-boot-starter#43]
This is for me...running mvn site causes issues with javadoc generation.  This additionally causes maven release plugin to fail.  This does not appear to be any issue with spring-boot-starter but rather how dependencies might be dealt with.  This place holder is so I don't forget this is an issue and hopefully can resolve before next release so the process is smoother.  If anyone else has any ideas on this, please do share.

**Comment by emacarron** [comment]
Seems javadoc is failing on may projects. Looks related to doclava... what about going to old javadoc?

**Comment by 805056790** [comment]
I got a problem , can anyone help and solve ??  http://stackoverflow.com/questions/36514622/mybatis-spring-boot-org-apache-ibatis-binding-bindingexception-invalid-bound-s

**Comment by emacarron** [comment]
@805056790 

Looks indeed like MyBatis is detecting your interface as a mapper. This is fixed in 1.1.0 which we will release inmediately.

Note that you will have to add the @Mapper annotation to your mapper so it can be identified by the auto-scanning.

Sorry!

**Comment by hazendaz** [comment]
Release javadoc issues have been resolved so closing this.

**Comment by hazendaz** [comment]
There is still some issue in release.  Before I released I knew there was a small issue with velocity file but that was not an issue with spring release.  Here, it failed as expected in same spot during site build but what additionally occurred was missing binaries in sonatype.  I had to run deploy with -Prelease to get the binaries out and get the release completed.  This may all in fact be related deploy and how sites work given this is a multi module build so will leave this open until next time.
